### PR TITLE
[IMP]made improvements, fix travis errors solve conflicts

### PR DESCRIPTION
**Pull request by Arpit-SerpentCS**

### CODE REVIEW

File: base_synchro/wizard/__init__.py [hunk 1]
```diff
@@ -1,3 +1,3 @@
-# -*- coding: utf-8 -*-
+#  coding: utf-8
```
Comment (JayVora-SerpentCS) [inline base_synchro/wizard/__init__.py]: remove the coding line from all py files.

File: label/security/.~lock.ir.model.access.csv# [hunk 2]
```diff
@@ -0,0 +1 @@
+,serpentcs,serpentcs-desktop,29.01.2018 11:18,file:///home/serpentcs/.config/libreoffice/4;
```
Comment (JayVora-SerpentCS) [inline label/security/.~lock.ir.model.access.csv#]: remove this.

File: base_synchro/wizard/__init__.py (hunk 1 above)
Comment (Arpit-SerpentCS) [reply base_synchro/wizard/__init__.py]: done

File: hr_applicant/models/__init__.py [hunk 3]
```diff
@@ -1,5 +1,6 @@
 # See LICENSE file for full copyright and licensing details.
 
+
```
Comment (JayVora-SerpentCS) [inline hr_applicant/models/__init__.py]: remove extra space

**Comment by JayVora-SerpentCS** [comment]
Make a decent PR please, if not made already

**Pull request closed without merging**